.github/workflows/ci.yml

```diff
@@ -118,14 +122,18 @@ jobs:
           extensions: dom, json, libxml, mbstring, pdo_sqlite, soap, xml, xmlwriter
           ini-values: assert.exception=1, zend.assertions=1
 
-      - name: Determine composer cache directory
-        id: determine-composer-cache-directory
-        run: echo "::set-output name=directory::$(./tools/composer config cache-dir)"
+      - name: Determine composer cache directory on Linux
+        if: matrix.os == 'ubuntu-latest'
+        run: echo "::set-env name=COMPOSER_CACHE_DIR::$(./tools/composer config cache-dir)"
+
+      - name: Determine composer cache directory on Windows
+        if: matrix.os == 'windows-latest'
+        run: ECHO "::set-env name=COMPOSER_CACHE_DIR::~\AppData\Local\Composer"
 
       - name: Cache dependencies installed with composer
         uses: actions/cache@v1
         with:
-          path: ${{ steps.determine-composer-cache-directory.outputs.directory }}
+          path: ${{ env.COMPOSER_CACHE_DIR }}
```

Comment:
I think that by making the path depend on the environment variable, then setting the environment variable depending on which operating system we are running on, the entire thing boils down to adding an `os` matrix, and we can skip the separate workflow file!

👍